qemu/tests/cfg/blockdev_commit_install.cfg

```diff
@@ -0,0 +1,40 @@
+- blockdev_commit_install:
+    type = blockdev_commit_install
+    virt_test_type = qemu
+    only Linux
+    need_install = yes
+    start_vm = no
+    image_name_image1 = "images/base_install"
+    boot_drive_stg = yes
+    medium = cdrom
+    installation = cdrom
+    kernel = vmlinuz
+    initrd = initrd.img
+    force_create_image_image1 = yes
+    image_aio = threads
+    unattended_delivery_method = cdrom
+    cdroms += " unattended"
+    index_enable = no
+    kill_vm = yes
+    shutdown_cleanly = no
+    storage_pools = default
+    storage_type_default = "directory"
+    storage_pool = default
+    snapshot_tags = sn1 sn2 sn3 sn4
+
+    image_name_sn1 = sn1
+    image_format_sn1 = qcow2
+
+    image_name_sn2 = sn2
+    image_format_sn2 = qcow2
+
+    image_name_sn3 = sn3
+    image_format_sn3 = qcow2
+
+    image_name_sn4 = sn4
+    image_format_sn4 = qcow2
+
+    device_tag = "image1"
+    rebase_mode = unsafe
+    mount_point = "/tmp"
```

Comment:
We can remove mount_point, as we don't write data to /tmp

Reply:
have updated.